app/views/about/help.md

```diff
@@ -4,6 +4,8 @@
 
 ## Main
 
+<div class="toc"></div>
+
```

Comment:
ah.. read more and ye shall find.

But does this mean we're leaving it up to content managers to put to ToC somewhere in the DOM (via markdown) to act as a placeholder?